de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/project/FileSystemChangeListener.java

```diff
@@ -317,15 +336,19 @@ public void fileMoved(
 
         path = makeAbsolutePathProjectRelative(path, project);
 
-        SPath newSPath = new SPath(project, path);
+        IReferencePointManager referencePointManager = resourceManager.getSession().
+            getComponent(IReferencePointManager.class);
+
+        SPath newSPath = new SPath(project.getReferencePoint(), path, referencePointManager);
 
         IPath oldParent = IntelliJPathImpl
             .fromString(virtualFileMoveEvent.getOldParent().getPath());
         IPath oldPath = oldParent.append(virtualFileMoveEvent.getFileName());
         IProject oldProject = getProjectForResource(oldPath);
 
         oldPath = makeAbsolutePathProjectRelative(oldPath, project);
-        SPath oldSPath = new SPath(oldProject, oldPath);
+
+        SPath oldSPath = new SPath(oldProject.getReferencePoint(), oldPath, referencePointManager);
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: A "NullPointerException" could be thrown; "oldProject" is nullable here. ([squid:S2259](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS2259))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=276&issues=AWajNu-SrChX7_AlOxX0&open=AWajNu-SrChX7_AlOxX0)

[ISSUE_KEY]: AWajNu-SrChX7_AlOxX0